pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+, unittest2
+, unidecode
+}:
+
+buildPythonPackage rec {
+  pname = "unicode-slugify";
+  version = "0.1.3";
+
+  propagatedBuildInputs = [ unidecode ];
```

Comment:
six should be here https://github.com/mozilla/unicode-slugify/blob/master/setup.py#L22

Reply:
fixed